cmd/wasm-client/main.go

```diff
@@ -345,27 +347,19 @@ func (w *wasmClient) InvokeRPC(_ js.Value, args []js.Value) interface{} {
 	method, ok := w.registry[rpcName]
 	if !ok {
 		log.Errorf("RPC method '%s' not found in registry", rpcName)
+
 		return js.ValueOf("rpc with name " + rpcName + " not found")
 	}
 
 	go func() {
-		defer func() {
```

Comment:
what was the original reason for adding this panic recovery log?

Reply:
I never actually ran into a panic here, but I thought it couldn't hurt to add these logs in the case that we hit that recovery state.